infrastructure/core/locals.tf

```diff
@@ -1,3 +1,3 @@
 locals {
-  subnet_address_spaces = cidrsubnets(var.core_address_space, 1, 2, 2)
+  subnet_address_spaces = cidrsubnets(var.core_address_space, 1, 3, 3, 3)
```

Comment:
probably need a larger webapp subnet – currently only 27 IPs. do we resize within the current address space or go for a /16 ?

Reply:
Yeah, Marcus advised we start smaller than that as that would be very wide. Perhaps if all the serve stuff has its own subnet, we make the core subnet smaller '2' - currently it's taking up most of the addresses and they aren't being used for anything